geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/util/ObjectPoolUtils.scala

```diff
@@ -0,0 +1,26 @@
+
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.spark.jts.util
+
+import org.apache.commons.pool.impl.GenericObjectPool
+
+// NB: This is a duplicate of a file in geomesa-utils
+trait ObjectPoolUtils[A] {
```

Comment:
can we actually just get rid of this class? I realize that in WKTUtils we switched to thread locals.

Reply:
I'll remove and see what happens.